src/module/s3.js

```diff
@@ -51,9 +56,15 @@ module.exports.S3 = ({ call }) => {
         ...(continuationToken === undefined && startAfter !== undefined ? { StartAfter: startAfter } : {}),
         ...(continuationToken === undefined ? {} : { ContinuationToken: continuationToken })
       });
-      result.push(...response.Contents);
-      continuationToken = response.NextContinuationToken;
-      isTruncated = response.IsTruncated;
+      if (stopAfter !== null && response.Contents[response.Contents.length - 1].Key >= stopAfter) {
+        result.push(...response.Contents.slice(0, response.Contents.findIndex((e) => e.Key.split('/')[0] > stopAfter)));
```

Comment:
since the keys are like: `2020-03-30T15:10:00.000Z/key3` and startAfter is `2020-03-30T15:10:00.000Z`. I cannot get an exact match to be returned unless I do a split.

Reply:
Why not? Just do a separate test?

Reply:
Or have a key that doesn't have the keyDIGIT part